### PR TITLE
Changed typo in html files and added test cases for comparison page

### PR DESCRIPTION
**Pull request by Eric-R-Liang**

### CODE REVIEW

File: Education_Pathways/templates/course.html [hunk 1]
```diff
@@ -60,7 +60,7 @@
             <a class="navbar-brand-top" href="/about/" style=valign:"center">About Us </a>
             <a class="navbar-brand-top" href="/filter" style=valign:"center">Filter</a>
             <a class="navbar-brand-top" href="/review/" style=valign:"center"> Review</a>
-            <a class="navbar-brand-top" href="/comparison/" style=valign:"center"> Comparison</a>
+            <a class="navbar-brand-top" href="/Comparison" style=valign:"center"> Comparison</a>
```
Comment (hsupeiyu12) [inline Education_Pathways/templates/course.html:63]: The format looks inconsistent. Maybe change 'Comparison' to '/comparison' in the index.py ?

Comment (Eric-R-Liang) [reply Education_Pathways/templates/course.html:63]: Would also require changing navbar.js and other relevant files. This formatting issue doesn't affect routing so no point in further modifications and testings IMO.

File: Education_Pathways/tests/test_app.py [hunk 2]
```diff
@@ -16,4 +16,9 @@ def test_course_details_endpoint():
 
     assert response.status_code == 200
 
+# Mandy - test
```
Comment (EmmaZhao0413) [inline Education_Pathways/tests/test_app.py]: maybe add some explanation as comment? @MandyHsu12

File: Education_Pathways/templates/course.html (hunk 1 above)
Comment (hsupeiyu12) [reply Education_Pathways/templates/course.html:63]: Sounds fair enough.

**Pull request merged**